theme/typography.ts

```diff
@@ -37,6 +37,7 @@ export const textStyle: Record<string, TypoToken> = {
     fontSize: 24,
     lineHeight: 31.2,
   },
+  "h2-sm-medium": { fontFamilyKey: "mont-500", fontSize: 24, lineHeight: 29 },
```

Comment:
Confirm with @lluisalleao that we need this preset, as I don't see it in the Figma:

<img width="754" height="846" alt="Image" src="https://github.com/user-attachments/assets/caaf05b8-b0e6-4835-9048-1903f105281f" />